### ISSUE TITLE
changeFieldControl doesn't allow multiple fields to be updated

### ISSUE THREAD

**Issue by mkalpana** [contentful/contentful-migration#216]
## Expected Behavior

Be able to change multiple editor interface in multiple fields together

## Actual Behavior

Error: Cannot save CT {content type} because it does not exist

## Steps to Reproduce

Executing the below migration script returns an error:
```
import { MigrationFunction } from 'contentful-migration';

export = function(migration) {
  // News Content Type with fields showAt and hideAt
  const news = migration.editContentType('news');
  ['showAt', 'hideAt'].forEach(fieldId => {
    news.changeFieldControl(fieldId, 'builtin','datePicker', {
      format: 'time'
    });
  });
} as MigrationFunction
```
I get the following error:

`Error: Cannot save CT news because it does not exist` 
## Context

I can replicate this with both `changeFieldControl` and deprecated `changeEditorInterface`.

Potentially similar to issue listed here: https://github.com/contentful/contentful-migration/issues/150

## Environment

-   **Node Version**: 10.15.0
-   **Package Manager Version**: 6.4.1
-   **Operating System**: macOS Mojave - 10.14.6
-   **Package Version**: 0.19.0

**Comment by thelucre** [comment]
Also seeing this issue: 

```
const block = migration.editContentType('block');

block.changeFieldControl('title', 'singleLine', 'builtin', {
    helpText: 'Only shown in the admin.',
});

block.changeFieldControl('isSquareAspect', 'singleLine', 'builtin', {
    helpText: 'If Yes, block will remain square after the mobile break. Useful for blocks with a Background image set.',
});
```

Displays this error when running the migration: 
`Error: Cannot save CT block because it does not exist`

**Comment by verys-jhuang** [comment]
I am also seeing this issue in contentful-migration 0.19.8 with the error:
`Error: Cannot save the content type (id: tag) because it does not exist`

Single field control changes work fine but multiple break it.

**Comment by Khaledgarbaya** [comment]
this is solved in `v1.0.2`
